### ISSUE TITLE
Add Plotly chart renderers

### ISSUE THREAD

**Issue by Eli-Berkow** [smartinsightsfromdata/rpivotTable#89]
I would like to track this SO question [R rpivotTable add Plotly chart renderers
](https://stackoverflow.com/questions/48440463/r-rpivottable-add-plotly-chart-renderers)

**Comment by smartinsightsfromdata** [comment]
@nicolaskruchten Any news on this issue?  Thanks in advance.

**Comment by nicolaskruchten** [comment]
My intention is to replace C3 with Plotly altogether but at the moment there isn't parity between the two because plotly.js does not yet support Stacked Areas (although it's being worked on). So the day that Plotly can be a drop-in replacement for C3 we can switch over.

We could switch over sooner but that would mean that anyone using Stacked Areas that switches would see breakage.

**Comment by Eli-Berkow** [comment]
@nicolaskruchten any chance of creating a separate branch in the meantime? I'm assuming by "drop-in replacement" it's an easy change but I might be way off.

**Comment by nicolaskruchten** [comment]
I don't have a lot of bandwidth to manage separate branches at the moment, unfortunately. Keep an eye on https://github.com/plotly/plotly.js/issues/1217 :)

**Comment by Eli-Berkow** [comment]
Thank you :)

**Comment by Eli-Berkow** [comment]
@nicolaskruchten looks like https://github.com/plotly/plotly.js/issues/1217 was just closed :)

**Comment by nicolaskruchten** [comment]
Yes I know ;) I can't commit a timeline for this, but at least there are now no new technical blockers.

**Comment by Eli-Berkow** [comment]
That's good. I'll keep an eye out, thanks.

**Comment by nicolaskruchten** [comment]
One step closer: PivotTable.js' plotly_renderers now has Area charts just like the c3_renderers
